examples/locks/locks.description

```diff
@@ -1,4 +1,4 @@
 // Ballerina locks are used for concurrency management. The encapsulating block of statements will acquire
 // the locks for each global or service level variable reference that is used within the block or for any
 // field access of an object or a record. When there is nested field access (Eg: record within a record)
-// all the filed accesses will be locked.
+// all the field accesses of the records will be locked.
```

Comment:
This is somewhat unclear. Does this mean that locks of each individual record will be acquired? If so, shall we rephrase this so that the meaning is clearer?

Reply:
Addressed with 259c088